src/components/client/campaign-application/helpers/validation-schema.ts

```diff
@@ -0,0 +1,30 @@
+import * as yup from 'yup'
+
+import { name, phone, email } from 'common/form/validation'
+
+import {
+  CampaignApplicationFormDataSteps,
+  CampaignApplicationOrganizer,
+  Steps,
+} from './campaignApplication.types'
+
+const organizer: yup.SchemaOf<CampaignApplicationOrganizer> = yup
+  .object()
+  .shape({
+    firstName: name.required(),
```

Comment:
Organizer name will be a single string. Checkout the https://dev.podkrepi.bg/swagger#/campaign-application/CampaignApplication.create schema

Reply:
Also - these values (name, email and phone ) should be prefilled from the current user. Please create a task for this over at https://github.com/users/gparlakov/projects/1/views/1

Reply:
> Organizer name will be a single string. Checkout the https://dev.podkrepi.bg/swagger#/campaign-application/CampaignApplication.create schema

Done